main.js

```diff
@@ -0,0 +1,101 @@
+
+//Show and Hide The news Sections
+var controlElement = document.querySelector('#show');
+controlElement.addEventListener('click', showHide);
+var content = document.querySelector('#mybutton');
+function showHide(event) {
+    event.preventDefault();
+    if (content.className === "hide") {
+        content.className = "";
+    }
+    else  {
+        content.className = "hide";
+    }
+}
+
+   
+
+
+//Form Selectors //
+
+var userEmail = document.querySelector('#useremail');
+var userName = document.querySelector('#username');
+var userPhone = document.querySelector('#userphone');
+
+var sbumitButton = document.querySelector('#submit-input');
+sbumitButton.addEventListener('click', submitForm);
+
+function submitForm(event) {
+ event.preventDefault();
+    verifyUserEmail();
+    verifyUserName();
+    verifyUserPhone();
+    if (verifyUserEmail() && verifyUserName() && verifyUserPhone()) {
+        userEmail.value = "";
+        userName.value = "";
+        userPhone.value = "";
+        alert('Thanks');
+       
+}
+}
+
+function verifyUserName() {
+    if (userName.value.length <= 0 ) {
+        userName.style.backgroundColor = "red";
+    }
+    else {
+        userName.style.backgroundColor = "white";
+        return true;
+    }
+}
+
+function verifyUserPhone() {
+    if ( userPhone.value.length < 11 && userPhone.value.length > 0 ) {
+        userPhone.style.backgroundColor = "white";
+        return true;
+    }
+    else {
+        userPhone.style.backgroundColor = "red";
+        return false;
+    }
+}
+
+function verifyUserEmail() {
+    if (userEmail.value.length <= 0 || textEmail.value.indexOf('@') < 0 ) {
+        userEmail.style.backgroundColor = "red";
+    }
+    else {
+        userEmail.style.backgroundColor = "white";
+        return true;
+    }
+}
+
+
+// AJAX REQUEST
+
+var xhrcontent = document.getElementsByClassName(content);
```

Comment:
very good!

Now as an extra challenge .. use `fetch` to make the AJAX call - see how much simpler you code becomes. https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch

(fetch doesn't work in older browsers but we don't care about that - http://caniuse.com/#feat=fetch)